h5p-bildetema-words-topic-image/src/components/TopicImageSVG/TopicImageSVG.module.scss

```diff
@@ -57,6 +57,7 @@
 
 .imageVertical {
   height: 100%;
+  max-height: calc(100vh - 5rem); // Compensate for mobile header height
```

Comment:
:smiles:

Reply:
If you know a better fix, then please tell me :crycat: